content/en/synthetics/api_tests/grpc_tests.md

```diff
@@ -0,0 +1,186 @@
+---
+title: gRPC Healthchecks
```

Comment:
I'm using Health Checking (https://github.com/grpc/grpc/blob/master/doc/health-checking.md) as the inspiration for title case in the page title!

```suggestion
title: GRPC Tests
```